xml/crm.dtd

```diff
@@ -65,7 +65,7 @@ Additionally, num_peers and have_quorum are used during the election process to
  * ccm_transition stores the membership instance from the ConsensusClusterMembership layer.
  * cib_feature_revision is the feature set that this configuration requires
 -->
-<!ELEMENT configuration (crm_config, nodes, resources, constraints)>
+<!ELEMENT configuration (crm_config, nodes, resources, constraints, alerts)>
```

Comment:
No need to change this. crm.dtd is the original xml schema before we had versioned rng's -- it's only ever used for validating configs that were generated pre-pacemaker-1.0 and have never been updated.